src/corefx/System.Globalization.Native/idna.cpp

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#include <stdint.h>
+#include <uchar.h>
+#include <unicode/uidna.h>
+
+const uint32_t AllowUnassigned = 0x1;
+const uint32_t UseStd3AsciiRules = 0x2;
```

Comment:
I would like us to have header files with just the public API that is expected to be consumed on the managed side. It makes it easier (IMHO) to compare/port the native declarations to the corresponding P/Invoke declarations.